app/models/manageiq/providers/storage_manager/storage_manager_mixin.rb

```diff
@@ -1,4 +1,4 @@
-module StorageManagerMixin
+module ManageIQ::Providers::StorageManager::StorageManagerMixin
```

Comment:
And @hsong-rh - with regard to the location of this file - previously you had it in app/models/mixins.  You seem to have moved it - it looks like all the mixing files are in that directory.  Should it go back there? (After renaming).